.github/workflows/check.sh

```
@@ -12,7 +12,7 @@ echo ===========================================================================
 # folder to ensure it is prepared again by meson.
 for wrap in subprojects/*.wrap; do
 	if ! git diff --quiet HEAD^ $wrap; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)